server/src/model/activist.go

```diff
@@ -272,10 +273,13 @@ SET
   street_address = :street_address,
   city = :city,
   state = :state,
+  address_updated = IF(street_address <> :street_address OR city <> :city OR state <> :state, NOW(), address_updated),
+  lat = :lat,
+  lng = :lng,
```

Comment:
i think the coords were purposefully not here before, b/c they are fields that are only set (and i believe only consumed) via the backend and are not exposed to users of the adb.

but i suppose this is fine, as long as the existing values are actually making it out to the frontend.

Reply:
looks like it's in the `ActivistJSON` struct, so should be fine.

Reply:
looks like i missed buildActivistJSONArray and CleanActivistData. thanks for helping me catch this!